app/views/discovered_hosts/show.html.erb

```diff
@@ -56,9 +59,12 @@
       </div>
     <% end %>
   </div>
+</div>
 
-<div class="panel-group" id="accordion" role="tablist" aria-multiselectable="true">
-    <div class="col-md-6">
+<div class="row">
+  <div class="col-md-12">
+    <div class="panel-group" id="accordion" role="tablist" aria-multiselectable="true">
+      <%= link_to(_("Expand All"),"#", :id => "expand_all", :class => "btn btn-default") -%>
       <% @categories.each_with_index do |val, index| %>
```

Comment:
@ik5 +1 on changing the position of the `Expand All` button. One question, can we have the `Expand All` button on the right side as it will maintain consistency?

Also, if you have time, can you fix the accordion, we use toggle for the `Expand all` button. If you open `Hardware` tab and then press `Expand All` button you will see that the `Hardware` tab will close.

Reply:
done for both :)